code/datums/map_config.dm

```diff
@@ -15,6 +15,7 @@
 
 	// Config actually from the JSON - should default to Box
 	var/map_name = "Box Station"
+	var/map_link = null //This is intentionally wrong, this will make it not link to webmap.
```

Comment:
box has its map unlike runtime

Reply:
The actual link for box is present, in the json for it, I believe this is just a default for when the json doesn't have a map_link or it fails to load entirely.

Reply:
KubeRoot is right, this is just the default value, if Box is the selected map, then this will be updated properly

Reply:
If the box is chosen as a fallback it should still have link to it

Reply:
If I set the default to box, then if, say, Metastation is selected, but Meta doesn't have a map_link defined in the json, then the map verb will provide a link to the box webmap, rather than letting the user know there is no correct map_link

Reply:
This is intentional and frankly unavoidable.